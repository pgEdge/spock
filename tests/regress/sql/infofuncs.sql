CREATE EXTENSION spock;

SELECT spock.spock_max_proto_version();

SELECT spock.spock_min_proto_version();

-- test extension version
SELECT spock.spock_version() = extversion
FROM pg_extension
WHERE extname = 'spock';

DROP EXTENSION spock;
