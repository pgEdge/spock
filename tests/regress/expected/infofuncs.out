CREATE EXTENSION spock;
SELECT spock.spock_max_proto_version();
 spock_max_proto_version 
-------------------------
                       4
(1 row)

SELECT spock.spock_min_proto_version();
 spock_min_proto_version 
-------------------------
                       3
(1 row)

-- test extension version
SELECT spock.spock_version() = extversion
FROM pg_extension
WHERE extname = 'spock';
 ?column? 
----------
 t
(1 row)

DROP EXTENSION spock;
